model/src/main/java/com/flipkart/flux/domain/Event.java

```diff
@@ -13,63 +13,123 @@
 
 package com.flipkart.flux.domain;
 
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
 import java.io.Serializable;
+import java.util.Date;
 
 /**
  * <code>Event</code> is the result of a {@link Task} execution.
  * It is to be posted back to the Flux execution engine once a worker has executed the task
- * 
+ *
  * @author Yogesh
```

Comment:
@regunathb - Do we really need these author tags considering everything is in git & this is open source?